sql/core/src/main/java/org/apache/spark/sql/sources/v2/DataSourceV2Options.java

```diff
@@ -49,4 +49,23 @@ public DataSourceV2Options(Map<String, String> originalMap) {
   public Optional<String> get(String key) {
     return Optional.ofNullable(keyLowerCasedMap.get(toLowerCase(key)));
   }
+
```

Comment:
>   /** Get a parameter as a boolean, falling back to a default if not set */

Reply:
Thanks.  Is it ok if I modify the comment a bit to this: 

  /**
   * Returns the integer value to which the specified key is mapped, 
   * or defaultValue if this map contains no mapping for the key.
   * The key match is case-insensitive
   */